pkg/repository/provider/unified_repo.go

```diff
@@ -568,9 +568,13 @@ func getStorageVariables(backupLocation *velerov1api.BackupStorageLocation, repo
 	result[udmrepo.StoreOptionOssRegion] = strings.Trim(region, "/")
 	result[udmrepo.StoreOptionFsPath] = config["fspath"]
 
-	if backupRepoConfig != nil {
-		if v, found := backupRepoConfig[udmrepo.StoreOptionCacheLimit]; found {
-			result[udmrepo.StoreOptionCacheLimit] = v
+	// Write all backupRepoConfig to results if not exists, otherwise error on conflict
```

Comment:
Sorry, I missed the question here https://github.com/vmware-tanzu/velero/pull/8581#issuecomment-2635349343.
Here in the unified_repo.go, we have `getStorageVariables` as a joint of translating user specified options to internal parameters, so we would accept only the options that are well defined in the internal system.

Therefore, I don't see any problem of the previous behavior or I don't see any necessity to change the existing behavior to accepting everything from backupRepoConfig

Reply:
Addressed in _96d1d74966c42abdb86e45873fe88779ef3cae2d_